### ISSUE TITLE
Construct delaunay network but one point didn't form triangles with other points

### ISSUE THREAD

**Issue by PengTao1997** [delfrrr/delaunator-cpp#15]

**Comment by delfrrr** [comment]
@PengTao1997 could please provide sample dataset and provide more details to reproduce the problem ?

**Comment by mourner** [comment]
Yeah, no way to reproduce unless there's a test case. But note that near-duplicate points are filtered out to be able to produce a valid triangulation.

**Comment by delfrrr** [comment]
@PengTao1997 does @mourner note answers your question?
